cloudify_libvirt/tests/test_common.py

```diff
@@ -100,9 +102,8 @@ def test_save_node_state(self):
                 makedirs
             ):
                 fake_file = mock.mock_open()
-                with mock.patch(
-                    '__builtin__.open', fake_file
-                ):
+                builtins_open = '__builtin__.open' if PY2 else 'builtins.open'
```

Comment:
Can you add this to cloudify-utilities-plugins-sdk _compat?

Reply:
Added to cloudify-utilities-plugins-sdk _compat and changed here to import the string